extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/security/PasswordPolicy.java

```diff
@@ -73,6 +73,20 @@
     int getMaximumAge() throws GuacamoleException;
 
     /**
+     * Returns the number of previous passwords remembered for each user. If
+     * greater than zero, users will be prohibited from reusing their past
+     * passwords.
```

Comment:
They'll be prohibited from reusing passwords in this history, right? This is a little vague and sounds like turning on this property ensures that a password can never be reused.